packages/edit-post/src/components/header/style.scss

```diff
@@ -249,3 +249,12 @@
 .components-popover.more-menu-dropdown__content {
 	z-index: z-index(".components-popover.more-menu__content");
 }
+
+.edit-post-header__preview-options {
```

Comment:
I copied the behavior of the site editor for now where we fade away to the button in zoom-out. I think that's not a great solution though, it's very adhoc and leave an empty space in the toolbar and also leaves a focusable but hidden button.

Reply:
I agree. Disabling it via aria-disabled seems like a better option. You can also hover the empty area and see the tooltip right now.